apps/site/src/app/(home)/sections/Countdown/Countdown.module.scss

```diff
@@ -0,0 +1,127 @@
+.paraboloid {
+	position: relative;
+	width: 100%;
+}
+
+.countdownWrapper {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	position: relative;
+}
+
+.descriptiveText {
+	top: 45px;
+	left: 20px;
+	position: absolute;
+	min-width: 150px;
+	transform: translateX(-50%);
+	text-align: center;
+}
+
+.countdownMaterial {
+	// margin-top: 10%;
+	position: absolute;
+	top: 0px;
+	left: 0px;
+	width: 100%;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
+
+@media only screen and (max-width: 400px) {
```

Comment:
Chore (repeating): rather than writing out all these media queries, we can use Bootstrap's mixins to handle that for us (refer to other stylesheets using bootstrap.media-breakpoint-up)

(thanks Taesung)

Reply:
(repeat): while this would be nice the current styling is acceptable, we could do fix this at a later date